apps/cli/README.md

```
@@ -1,20 +1,62 @@
-# apps/cli/.env.local (optional — also reads ../api/.env.local)
-FD_API_URL=http://localhost:3333/api/ls
-FD_WEB_URL=http://localhost:3000
-DISCORD_BOT_KEY=<same value as apps/api>
-FD_DEV_ORG=acme
+FD_LOCAL_ORG_SLUG=acme
+FD_PROD_ORG_SLUG=your-org
+```
+
+Then build the URL yourself from a created thread:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)